pages/ugm2024.md

```diff
@@ -24,7 +24,7 @@ The agenda will be announced in Spring 2024.
 ## Abstracts and Registration
 *Abstract and bursary application deadline: Friday, March 15, 2024*
 
-A call for abstracts will open February 1, 2024. We welcome abstracts from all areas relevant to MDAnalysis's work, including the use and development of open source software and tools for molecular simulations data analysis. 
+You may now submit an abstract using the [abstract submission form](https://forms.gle/usXuZVeM1iZiTDZY8). We welcome abstracts from all areas relevant to MDAnalysis's work, including the use and development of open source software and tools for molecular simulations data analysis. 
```

Comment:
add deadline for abstract submission

Reply:
It is also listed above this paragraph, but I've added the deadline after the submission form to make it even more clear.